src/DetailsView/components/requirement-view-title.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { GuidanceTags, GuidanceTagsDeps } from 'common/components/guidance-tags';
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+import { ContentPageComponent, HyperlinkDefinition } from 'views/content/content-page';
+import { ContentPanelButton, ContentPanelButtonDeps } from 'views/content/content-panel-button';
+
+export type RequirementViewTitleDeps = GuidanceTagsDeps & ContentPanelButtonDeps;
+
+export interface RequirementViewTitleProps {
+    deps: RequirementViewTitleDeps;
+    name: string;
+    guidanceLinks: HyperlinkDefinition[];
+    infoAndExamples: ContentPageComponent;
+}
+
+export const RequirementViewTitle = NamedFC<RequirementViewTitleProps>(
+    'RequirementViewTitle',
+    props => {
+        return (
+            <h1 className="requirement-view-title">
```

Comment:
should this class come from a scss file?

Reply:
Added requirement-view-title.scss